Library/Formula/docbook.rb

```diff
@@ -45,7 +45,14 @@ class Docbook < Formula
 
   def install
     (etc/"xml").mkpath
-    system "xmlcatalog", "--noout", "--create", "#{etc}/xml/catalog"
+
+    # only create catalog file if it doesn't exist already to avoid content added
+    # by other formulas to be removed
+    unless File.file?("#{etc}/xml/catalog")
```

Comment:
Potentially worth noting is that if this file does exist locally, the bottle installation won't overwrite it. Installs as `catalog.default` instead.